portable-atomic-util/tests/arc.rs

```diff
@@ -257,8 +257,12 @@ mod alloc_tests {
         assert_eq!(unsafe { &*ptr }, "foo");
         assert_eq!(arc, arc2);
 
-        // TODO: CoerceUnsized is needed to cast to Arc<dyn ..>
-        // let arc: Arc<dyn Display> = Arc::new(123);
+        // XXX INJECT NEGATIVE SIGN - SEE IF CI FAILS AS EXPECTED WITH THIS INCORRECT VALUE
+        // XXX TODO NEED TO CHANGE THIS INTO A POSITIVE NUMBER FOR CI TEST TO SUCCEED
+        #[cfg(portable_atomic_unstable_coerce_unsized)]
+        let arc: Arc<dyn Display> = Arc::new(-123);
+        // TODO: This is a workaround in case CoerceUnsized is not available - remove once this is no longer needed
+        #[not(cfg(portable_atomic_unstable_coerce_unsized))]
         let arc: Arc<dyn Display> = Arc::from(Box::new(123) as Box<dyn Display>);
```

Comment:
Need to update like this once we see that CI shows failure as expected with this incorrectly negated value:

```suggestion
        #[cfg(portable_atomic_unstable_coerce_unsized)]
        let arc: Arc<dyn Display> = Arc::new(123);
        // TODO: This is a workaround in case CoerceUnsized is not available - remove once this is no longer needed
        #[not(cfg(portable_atomic_unstable_coerce_unsized))]
        let arc: Arc<dyn Display> = Arc::from(Box::new(123) as Box<dyn Display>);
```

@taiki-e I see several other tests with "TODO: CoerceUnsized is needed to cast to Arc<...>" - would it be enough to simply updated all of these cases to test with `cfg(portable_atomic_unstable_coerce_unsized)` to test this feature?